README.md

```diff
@@ -126,56 +129,154 @@ Disk space: 50 GB required for the docker, 25 GB available free space in the doc
+
+
+<h2> 💻 User Feedback </h2>
+
+Users can provide feedback via the 👍 and 👎 icons to the answers shown in the results page. 
+
+To use the feedback to fine-tune your Reader model
+
+1. Get the feedback data:
+  
+  ```sh
+    curl -X 'GET' \
+  'http://localhost:50059/feedbacks?application=reading&application=qa&_format=primeqa' \
+  -H 'accept: application/json' > feedbacks.json
+  ```
+
+2. Follow the instructions on how to finetune a PrimeQA reader with custom data [here](https://github.com/primeqa/primeqa/tree/main/examples/custom_mrc#finetuning-using-feedback-data). Generally, the finetuning would start with the model used when collecting the feedback data as specified in the `Model` field under `Reader` settings in the `Reading` and/or `QuestionAnswering` UI.
+
+3. To deploy the finetuned model, follow the instructions [here](#custom-mrc).
+
+
+
+<h2 id="troubleshooting"> 🤨 Troubleshooting</h2>
+
+a. If the UI is not loading properly or a field is blank, please try these quick steps:
+   - clear the browser cache and retry
+   - restart the containers by running `terminate.sh` and then `launch.sh`
+
+b. To view the logs, use the docker logs command, for example:
+
+    ```
+    docker logs primeqa-ui
+    docker logs primeqa-orchestrator
+    docker logs primeqa-services
+    ```
+
+<h2> 🤨 Frequently Asked Questions (FAQs) </h2>
 
-<h4> 2. How to retrieve feedback data for finetuning my reader model? </h4>
+1. How do I switch to a different PrimeQA Reader model from the Huggingface model hub ?
 
-To download feedback data, please refer to the FAQs section [here](https://github.com/primeqa/primeqa-orchestrator#1-how-do-i-get-feedbacks-to-fine-tune-my-reader-model-).
+    Paste the model name from the [Huggingface model hub](https://huggingface.co/PrimeQA) into the  `Model` field under `Reader` settings in the `Reading` and/or `QuestionAnswering` UI.
+ 
+2. <a id="custom-mrc"></a>How do I use my custom model for reader in `Reading` or `QA` application?
 
-<h4> 3. How to fine tune a reader model? </h4>
+    By default the reader initializes the `PrimeQA/nq_tydi_sq1-reader-xlmr_large-20221110` from the Huggingface model hub. 
 
-1. Download necessary feedback data to fine tune your reader model as per [FAQs #2](https://github.com/primeqa/create-primeqa-app#2-how-to-retrieve-feedback-data-to-fine-tune-my-reader-model-).
+    To use your own reader model, place your model in a directory under `primeqa-store/model` directory.  To point to your model from the UI, navigate to  `Application Settings`, scroll down to `Reader Settings` and to `Model` and set it to `/store/model/<model-dir>`, replace `model-dir` with the name of the directory containing the model files.
```

Comment:
@bhavani105 Sorry to be annoying but I think it is `primeqa-store/models` and not `primeqa-store/model`